apps/dhparam.c

```diff
@@ -364,10 +374,14 @@ int dhparam_main(int argc, char **argv)
         const BIGNUM *q;
         DH_get0_pqg(dh, NULL, &q, NULL);
         if (outformat == FORMAT_ASN1) {
+        /* TODO(3.0): put this back and get it working */
+        #if 0
```

Comment:
PEM functions aren't deprecated but i2d are?

Reply:
Seems so.  @mattcaswell, @levitte and I had a discussion about what to deprecate and what to leave alone.